sem-version

```diff
@@ -94,7 +94,18 @@ version::change_scala() {
 }
 version::change_erlang() {
   version::change "change-erlang-version"
-
+  return $?
+}
+version::change_firefox() {
+  local language=""
+  case "${language_version}" in
+    "52")language="esr-prev";;
+    "68")language="esr";;
+    "78")language="esr-new";;
+    *)echo "Please choose one of 52, 68, 78";exit 1;;
+  esac
+  language_version="$language"
+  version::change "change-firefox-version"
```

Comment:
The usage of the word `language` for handling non-language things makes this code hard to read and maintain. We need to improve this.

Reply:
Changed `language` to `ff_version`.

Reply:
This one is still ambiguous `language_version="$ff_version"`. Let's refactor the script if necessary for long term clarity and maintainability.